pkg/envoy/ads/response_benchmark_test.go

```diff
@@ -187,7 +187,6 @@ func BenchmarkSendXDSResponse(b *testing.B) {
 					b.Fatalf("Failed to send response: %s", err)
 				}
 			}
-			b.StopTimer()
```

Comment:
you intend to remove this?

Reply:
Yes. It's not necessary in this case since it's the end of the test.